v8.go

```diff
@@ -77,14 +77,19 @@ type Snapshot struct{ data C.StartupData }
 
 func newSnapshot(data C.StartupData) *Snapshot {
 	s := &Snapshot{data}
-	runtime.SetFinalizer(s, (*Snapshot).release)
```

Comment:
Sorry, I didn't know about [method expressions](https://golang.org/ref/spec#Method_expressions). I can revert that, the main reason I did that is because I wanted to put some debug statements to print information to console to verify that the finalizers were getting called. I am not sure if they are getting called or not.

For example: 

```go
func main() {
  ctx := NewIsolate().NewContext()
  // Doing something...
  
  // Done with ctx, set to nil and it should trigger the finalizer to run. Also, ctx holds a pointer to the isolate and the isolate doesn't get cleaned up either.
  ctx = nil
}
```

Am I understanding this correctly? There are couple of reasons why it is happening, and will detail it once I do a little more digging.